firecrawl/cmd/scrape.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/mendableai/firecrawl-go"
+)
+
+func Scrape(ctx context.Context, apiKey, url string) (string, error) {
+	apiKey = strings.TrimSpace(apiKey)
+	if apiKey == "" {
+		return "", fmt.Errorf("API key is required")
+	}
+
+	if url == "" {
+		return "", fmt.Errorf("URL is required")
+	}
```

Comment:
Can this validation be moved to `main.go`? It seems like we would have to do this for each command we add, so it would be better suited in a more centralized location.